### PR TITLE
Pepper-2020-100

### PR DESCRIPTION
**Pull request by CarolinHu**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

A pretty straightforward, noun-only list. Two questions right away:

- The list has both supper and dinner, both were mapped to DINNER (SUPPER) for now. Is one more prototypical?
- We have both SPECTACLES and GLASSES as concept sets and I mapped the item in this list to SPECTACLES, but I don't quite see the difference between the two sets. Are they distinguished in any way?

<details><summary>And here's concepticon check:</summary>
<p>

                          4  Pepper-2020-100-3          3  backpack
                          8  Pepper-2020-100-7          7  bicycle pump
                         19  Pepper-2020-100-18        18  dairy cow
                         33  Pepper-2020-100-32        32  flea market
                         38  Pepper-2020-100-37        37  gold ring
                         39  Pepper-2020-100-38        38  hand brake
                         40  Pepper-2020-100-39        39  handbag
                         45  Pepper-2020-100-44        44  keyword
                         51  Pepper-2020-100-50        50  mail box
                         62  Pepper-2020-100-61        61  paddle wheel
                         67  Pepper-2020-100-66        66  railway
                         79  Pepper-2020-100-78        78  stone bridge
                         87  Pepper-2020-100-86        86  toilet paper
                         90  Pepper-2020-100-89        89  toolbox
                         96  Pepper-2020-100-95        95  water pump
                         99  Pepper-2020-100-98        98  windmill
1833                     20  Pepper-2020-100-19        19  dinner
1833                     82  Pepper-2020-100-81        81  supper
Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK
</p>
</details>

@Kristina-Pianykh, may I ask you to moderate and take a look?

### CODE REVIEW

**Comment by Kristina-Pianykh** [comment]
Absolutely. Could you @AnnikaTjuka and @Schweikhard have a look at this?
In respect to GLASSES and SPECTACLES, one of them should be retired as to my knowledge there is no difference between these terms except 'spectacles' is more old-fashioned. As far as I remember, the conception policy is to retire the latest added concept set (which is GLASSES).

'Dinner' seems to refer not only to the latest meal of the day but also to formal events (e.g. guests are invited to dinner and not supper). In other words, I'd say it has a broader meaning than 'supper'. Having the current definition of DINNER (SUPPER) in mind, I'd map only 'supper' then.

**Comment by AnnikaTjuka** [comment]
Sure! I'll submit my review later today.

File: concepticondata/conceptlists/Pepper-2020-100.tsv [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Pepper-2020-100-10	10	boy	1366	BOY
+Pepper-2020-100-11	11	bracelet	1754	BRACELET
+Pepper-2020-100-12	12	breakfast	1322	BREAKFAST
+Pepper-2020-100-13	13	carpenter	1042	CARPENTER
+Pepper-2020-100-14	14	chieftain	301	CHIEFTAIN
+Pepper-2020-100-15	15	cock/rooster	1511	ROOSTER
+Pepper-2020-100-16	16	collarbone	1994	COLLARBONE
+Pepper-2020-100-17	17	cookhouse	334	COOKHOUSE
+Pepper-2020-100-18	18	dairy cow		
+Pepper-2020-100-19	19	dinner	1833	DINNER (SUPPER)
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Pepper-2020-100.tsv:20]: I agree with @Kristina-Pianykh 's comment and would only map 'supper'.

**Comment by CarolinHu** [comment]
Thank you all for reviewing, I unmapped dinner and changed a typo. Is there anything else or can this be merged?

**Comment by Kristina-Pianykh** [comment]
It all looks good and you can merge the PR! Resolving the conflict between GLASSES and SPECTACLES and retiring will be done then in a separate PR. I'm not sure if there is an automated way to do this so that the lists mapped to the retired concept set are moved in the proper manner. At least I'm not aware of such a method.

**Pull request merged**